lib/streamlit/elements/camera_input.py

```diff
@@ -11,10 +11,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from __future__ import annotations
```

Comment:
nit: is this being used? I wouldn't expect both it and imports from the `typing` module / would probably prefer tweaking these type annotations later.